schema-dex-agg.graphql

```diff
@@ -469,6 +507,9 @@ type Account @entity {
   " { Account address } "
   id: Bytes!
 
+  " All historical trade volume in USD "
+  cumulativeTotalVolumeUSD: BigDecimal!
```

Comment:
in the rest of the file you use `cumulativeVolumeUSD` and not `cumulativeTotalVolumeUSD` is there a diff?

Reply:
Just leaving cumulativeVolumeUSD makes more sense. I updated to fix this.